lib/generate-sitemap.js

```diff
@@ -0,0 +1,87 @@
+const url = require('url');
+const sitemapGenerator = require('sitemap');
+
+const register = require('../fixtures/register.json');
+const Fixture = require('../lib/model/Fixture.js');
+
+module.exports = function generateSitemap(options) {
+  const urls = getStaticUrls()
+    .concat(getRegisterUrls());
+  
+  const sitemap = sitemapGenerator.createSitemap({
+    hostname: url.resolve(options.url, '/'),
+    urls: urls
+  });
+
+  return sitemap.toString();
+};
+
+function getStaticUrls() {
+  return [
```

Comment:
Maybe have an object like this that is translated? Would make the code a little shorter and catchier.

```js
{
  '/': 'daily',
  '/fixture-editor': 'monthly',
  ...
}
```

Reply:
But defining each url object on its own is more descriptive (e.g. it's clear that 'monthly' is the change frequency) and it allows for additional properties for specific urls